core/bourbon/settings/_bourbon-get-setting.scss

```diff
@@ -0,0 +1,13 @@
+/// Return a Bourbon setting.
+///
+/// @argument {string} $setting
+///
+/// @example scss
+///   _bourbon-get-setting(rails-asset-pipeline)
+///
+/// @access private
+
+@function _bourbon-get-setting($setting) {
+  $settings: map-merge($_bourbon-defaults, $bourbon);
```

Comment:
I must say I don’t know what is the fastest options between the two following expressions:

```scss
@return map-get(map-merge($_bourbon-defaults, $bourbon), $setting);
```

```scss
@return map-get($bourbon, $setting) or map-get($_bourbon-defaults, $setting);
```

It would be interesting to ask someone who knows or to run a benchmark.